composer/trainer/trainer.py

```diff
@@ -622,6 +622,17 @@ class Trainer:
             If a callable, it should take one argument which is the state_dict. The callable is free to arbitrarily modify
             the state_dict before it is loaded.
 
+            (default: ``None``)
+        load_exclude_algorithms (List[str], optional): A list of algorithm names to exclude from loading.
```

Comment:
`exclude_algorithms_on_load`? or is it too long?

Reply:
would prefer to prefix with `load_` since all loading relating arguments are prefixed similarly